west.yml

```diff
@@ -11,4 +11,4 @@ manifest:
     - name: nrfxlib
       remote: ncs
       repo-path: sdk-nrfxlib
-      revision: a7dd3441ed55ee05234ed15c44c84243229c9876
+      revision: pull/1172/head
```

Comment:
This is already merged please update to `5bc2f4c90989639bf4d6c2275c241523cdfe30c2`